radixdlt/src/main/java/org/radix/network2/addressbook/PeerManager.java

```diff
@@ -125,25 +120,24 @@ public void run() {
 	}
 
 	@Inject
-	PeerManager(PeerManagerConfiguration config,
-	            AddressBook addressbook,
-	            MessageCentral messageCentral,
-	            Events events,
-	            BootstrapDiscovery bootstrapDiscovery,
-	            SecureRandom rng,
-	            @Named("self") EUID self,
-	            LocalSystem localSystem,
-	            Interfaces interfaces,
-	            RuntimeProperties properties, Universe universe) {
+	PeerManager(
+		PeerManagerConfiguration config,

```

Comment:
"self" and "events" removed, and reformatted to use Radix standard lots-of-arguments formatting